src/ripple/protocol/impl/RippleAddress.cpp

```diff
@@ -59,23 +59,21 @@ static uint128 PassPhraseToKey (std::string const& passPhrase)
 
 static Blob getPublicKey (openssl::ec_key const& key)
 {
-	Blob result (33);
-	
-	key.get_public_key (&result[0]);
-	
-	return result;
+    Blob result (33);
```

Comment:
Not sure if I agree that it adds anything. ec_key public keys that we use are always 257 bits. That's never going to change. So what do we gain by enshrining it in a constant? A comment would suffice.